### PR TITLE
fix: add support for input types to `getArgs`

### LINKED ISSUE #434: `getArgs` doesn't work for input types
```graphql
type Query {
  blog: Blog
}

type Blog {
  comments(where: FilterInput!): Comment[]
}

input FilterInput {
  author: String
}
```

```graphql
query {
  blog {
    comments(where: { author: "smeijer" }) {
      body
    }
  }
}
```

```js
db.blog.astToQuery(ast, {
  embody({ body, getArgs }) {
    // `args.where` is always undefined
    const args = getArgs('security');
  }
});
```

### PR DESCRIPTION
**Pull request by smeijer**
fixes #434

### CODE REVIEW

**Comment by smeijer** [comment]
Someone might want to extend this, and add support for nested `input` types as well. This can "easily" be done adding some recursion to my change.

**Comment by smeijer** [comment]
Made it recursive as well.

File: lib/graphql/lib/astToBody.js [hunk 1]
```diff
@@ -10,21 +10,29 @@ export default function astToBody(ast) {
...
 }
 
+function extractArguments(args) {
+  if (!Array.isArray(args) || args.length === 0) {
+    return {};
+  }
+
+  return args.reduce((acc, field) => {
+    acc[field.name.value] =
+      field.value.kind === 'ObjectValue'
```
Comment (theodorDiaconu) [inline lib/graphql/lib/astToBody.js:20]: Can you please add a descriptive comment for this?

Comment (smeijer) [reply lib/graphql/lib/astToBody.js:23]: done. Sorry for the delay.

Comment (smeijer) [reply lib/graphql/lib/astToBody.js:23]: @theodorDiaconu, does the added comment work for you?

**Pull request merged**